src/components/LcButton/__tests__/LcButton.spec.ts

```diff
@@ -0,0 +1,337 @@
+import { mount } from '@vue/test-utils'
+import LcButton from '../LcButton.vue'
+import { vColor, vFontWeight, vSize, vVariant } from '../../../validators/LcButton'
+
+let wrapper: any
+
+afterEach(() => {
+  wrapper?.unmount()
+})
+
+describe('LcButton', () => {
+  it('is a Vue instance', () => {
+    wrapper = mount(LcButton)
+    expect(wrapper.vm).toBeTruthy()
+  })
+
+  it('Should be a loader button', () => {
+    wrapper = mount(LcButton, { props: { loader: true } })
+
+    expect(wrapper.html()).toMatchSnapshot()
```

Comment:
ce serait peut-être pas mal de vérifier que la div du loader existe ? Hier en faisant de la review j'ai vu un snapshot sur leonardo "should display list of result" qui avant la pr ne contenait aucun résultat 😬  donc je pense que c'est pas mal d'assurer ses arrières quand on utilise le snapshot, comme pour les classes juste en-dessous